src/viam/app/data_client.py

```diff
@@ -338,6 +339,44 @@ async def get_latest_tabular_data(
             return None
         return response.time_captured.ToDatetime(), response.time_synced.ToDatetime(), struct_to_dict(response.payload)
 
+    async def export_tabular_data(
+        self, part_id: str, resource_name: str, resource_subtype: str, method_name: str, start_time: Optional[datetime] = None, end_time: Optional[datetime] = None
+    ) -> List[ExportTabularDataResponse]:
+        """Obtain unified tabular data and metadata from the specified data source.
+
+        ::
+
+            tabular_data = await data_client.export_tabular_data(
+                part_id="<PART-ID>",
+                resource_name="<RESOURCE-NAME>",
+                resource_subtype="<RESOURCE-SUBTYPE>",
+                method_name="<METHOD-NAME>",
+                start_time="<START_TIME>"
+                end_time="<END_TIME>"
+            )
+
+            print(f"My data: {tabular_data}")
+
+        Args:
+            part_id (str): The ID of the part that owns the data.
+            resource_name (str): The name of the requested resource that captured the data.
+            resource_subtype (str): The subtype of the requested resource that captured the data.
+            method_name (str): The data capture method name.
+            start_time (datetime): Optional start time for requesting a specific range of data.
+            end_time (datetime): Optional end time for requesting a specific range of data.
+
+        Returns:
+            List[ExportTabularDataResponse]: The unified tabular data and metadata.
+
+        For more information, see `Data Client API <https://docs.viam.com/appendix/apis/data-client/>`_.
+        """
+
+        interval=CaptureInterval(start=datetime_to_timestamp(start_time), end=datetime_to_timestamp(end_time))
+        request = ExportTabularDataRequest(
+            part_id=part_id, resource_name=resource_name, resource_subtype=resource_subtype, method_name=method_name, interval=interval
+        )
+        return await self._data_client.ExportTabularData(request, metadata=self._metadata)
```

Comment:
`ExportTabularDataResponse` has a lot of data in it that is maybe not needed? Which fields on the response would the user actually care about? Is it only the `payload`? Or also `tags`? 

`payload` is a proto.Struct, which has really bad UI. We should avoid using that where we can, so we should determine what's actually important to return

Reply:
There's a note in the scope to optionally only return the payload in the future. As for our comment regarding proto.Struct, probably best for us to talk in person, so I'll wanter over to your area again at some point!